src/app/pages/password.page.ts

```diff
@@ -5,12 +5,103 @@
  */
 
 import { Component } from '@angular/core';
+import { DemoTabData } from '../components/demo.component';
 
 @Component({
   selector: 'app-password',
   template: `
     <h1>Password</h1>
-    <not-ready name="Password"></not-ready>
+    <demo [tabs]="demo1">
+      <h3>Basic</h3>
+    </demo>
+
+    <demo [tabs]="demo2">
+      <h3>password with label</h3>
+    </demo>
+
+    <demo [tabs]="demo3">
+      <h3>password with label</h3>
+    </demo>
+
+    <demo [tabs]="demo4">
+      <h3>All type of paswords</h3>
```

Comment:
What do you mean by `All type of paswords` (there is a typo in password) 

Do you mean `Containers for password` ?